### ISSUE TITLE
What should DNN's training data look like in the end?

### ISSUE THREAD

**Issue by zhouhao-learning** [ExaScience/process-plate#4]
Because of the privatization of your data, I understand, but one of my questions is, I don't know what the training data really looks like, a CSV file or image data? If it's a CSV file, what features should be included? Is it convenient to provide an example of training data, even if it's just one data, even if it's a fake number? It doesn't matter.
Thank you very much!!! I believe this is also useful for other people.

**Comment by roelWuyts** [comment]
The input data are images (more specifically, images coming out of high content imaging runs, e.g. images of wells with hundreds of cells per image). The CellProfiler pipeline extracts (calculates) features from these images (e.g. number of cells, average cell size, ...). elPlate can be used to do that for many wells. The example pipeline is meant to give an idea, but I have indeed not passed example data because we do not own it (I will try to get permission to add data for some wells). The output result of the pipeline is a CSV file with the measured data.
Note that elPlate does not do any machine learning, it merely extracts the features from cells.

**Comment by zhouhao-learning** [comment]
@roelWuyts 
Ok, thank you!!!
If the input data is an image, then what do I use to divide the label of the image? Supervised learning should correspond to a real label. Finally, how do I predict whether an image is active or inactive?

**Comment by roelWuyts** [comment]
Sorry, can't help you there since I did not do the machine learning, merely making sure the data was there. You should contact our colleagues from the KU Leuven for this (if this is still regarding the results from the paper).
